models/merkle_path.go

```diff
@@ -106,6 +106,7 @@ func (p *MerklePath) Child(right bool) (*MerklePath, error) {
 	}, nil
 }
 
+// TODO: can this ever really error?
```

Comment:
oops, I'll remove this. I think this can be refactored but that belongs in a future PR